versions/3.2.0.md

```diff
@@ -3292,6 +3292,7 @@ Field Name | Type | Description
 <a name="oauthFlowsPassword"></a>password| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Resource Owner Password flow 
 <a name="oauthFlowsClientCredentials"></a>clientCredentials| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Client Credentials flow.  Previously called `application` in OpenAPI 2.0.
 <a name="oauthFlowsAuthorizationCode"></a>authorizationCode| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Authorization Code flow.  Previously called `accessCode` in OpenAPI 2.0.
+<a name="oauthFlowsDeviceAuthorization"></s>deviceAuthorization| [Oauth Flow Object](#oauthFlowObject) | Configuration for the OAuth Device Authorization flow.
```

Comment:
Small typo, should be `OAuth Flow Object` not `Oauth Flow Object`

Reply:
I just want to say that I read this a few days ago, and I mean to fix it, but it keeps slipping my mind. >_< I am so sorry.

Reply:
Fixed. I apologize for the delay.